crates/transaction-pool/src/validate/eth.rs

```diff
@@ -279,14 +279,18 @@ where
 
         // Drop non-local transactions with a fee lower than the configured fee for acceptance into
         // the pool.
-        if !self.local_transactions_config.is_local(origin, transaction.sender_ref()) &&
-            transaction.is_eip1559() &&
-            transaction.max_priority_fee_per_gas() < self.minimum_priority_fee
-        {
-            return TransactionValidationOutcome::Invalid(
-                transaction,
-                InvalidPoolTransactionError::Underpriced,
-            )
+        if !self.local_transactions_config.is_local(origin, transaction.sender_ref()) {
+            match (transaction.max_priority_fee_per_gas(), self.minimum_priority_fee) {
+                (Some(max_priority_fee), Some(min_priority_fee)) => {
+                    if max_priority_fee < min_priority_fee {
```

Comment:
isn't this the same as 

```
transaction.max_priority_fee_per_gas() < self.minimum_priority_fee
```

Reply:
If max_priority_fee_per_gas return None the result will be also true